lib/ain-ocean/src/api/pool_pair/path.rs

```diff
@@ -275,6 +275,55 @@ fn all_simple_paths(
     Ok(paths)
 }
 
+fn get_dex_fees_pct(
+    pool_pair_info: PoolPairInfo,
+    from_token_id: &String,
+    to_token_id: &String,
+) -> Option<EstimatedDexFeesInPct> {
+    let PoolPairInfo {
+        id_token_a,
+        id_token_b,
+        dex_fee_in_pct_token_a,
+        dex_fee_out_pct_token_a,
+        dex_fee_in_pct_token_b,
+        dex_fee_out_pct_token_b,
+        ..
+    } = pool_pair_info;
+
+    let token_a_direction = if id_token_a == *from_token_id {
+        "in"
+    } else {
+        "out"
+    };
```

Comment:
Could clean it up with a Direction enum

Reply:
fixed [662a9e5](https://github.com/DeFiCh/ain/pull/2925/commits/662a9e53cc4f781e705b6092bdfbb41495379175)